neurom/apps/__init__.py

```diff
@@ -28,19 +28,21 @@
 
 '''Helper code for neurom applications'''
 import yaml
+import logging
 
 from neurom.exceptions import ConfigError
 
 
 def get_config(config, default_config):
     '''Load configuration from file if in config, else use default'''
-    if config:
-        try:
-            with open(config, 'r') as config_file:
-                return yaml.load(config_file)
-        except (yaml.reader.ReaderError,
-                yaml.parser.ParserError,
-                yaml.scanner.ScannerError) as e:
-            raise ConfigError('Invalid yaml file: \n %s' % str(e))
-    else:
-        return default_config
+    if not config:
+        logging.warn('Using default config: {}'.format(default_config))
```

Comment:
I used the "warn" level as I wanted it to be printed even when the user is not passing the verbose option